scalac-scoverage-plugin/src/main/scala/scoverage/report/ScoverageXmlWriter.scala

```diff
@@ -96,5 +103,7 @@ class ScoverageXmlWriter(sourceDir: File, outputDir: File, debug: Boolean) {
     </package>
   }
 
+  private def relativeSource(src: String): String = IOUtils.relativeSource(src, formattedSourcePaths)
```

Comment:
And if `formattedSourcePaths` is used only here, maybe move it's implementation to `IOUtils.relativeSource` to avoid this duplication?